apps/academy-wep/components/NewsletterSubscription.tsx

```
@@ -0,0 +1,215 @@
+								<h3 className="font-semibold text-primary">
+									{toastMessage.title}
+								</h3>
+								<p className="text-sm text-slate-600">
+									{toastMessage.description}
+								</p>
+							</div>
+							<button
+								onClick={() => setShowToast(false)}
+								className="text-slate-400 hover:text-slate-600"
+								aria-label="Close notification"
+							>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)